client/components/track-input-changes/test/index.jsx

```diff
@@ -50,7 +50,7 @@ describe( 'TrackInputChanges#onNewValue', function() {
 	} );
 
 	beforeEach( function() {
-		for ( let spy in spies ) {
+		for ( const spy in spies ) {
```

Comment:
Note: There's a really tricky nuance that makes this line [valid](https://stackoverflow.com/questions/41067790/why-does-const-work-in-some-for-loops-in-javascript). Usually, using `const` doesn't work very well inside for loop declarations.